### PR TITLE
test: Comparing rpc and tonic performance

### PR DESCRIPTION
**Pull request by Tricky511**
rpc:
examples/rpc_server.rs 
examples/rpc_client_async.rs

tonic:
examples/grpc_server.rs
examples/grpc_client.rs

### CODE REVIEW

**Comment by uran0sH** [comment]
What's the result of comparision

**Comment by Tricky511** [comment]
> What's the result of comparision

The speed is 1/3~1/4 of the Tonic

File: rpc/src/connection.rs [hunk 1]
```diff
@@ -449,7 +447,7 @@ impl ServerConnection {
                     return Err(e.into());
                 }
             }
-            sleep(Duration::from_millis(100));
```
Comment (luanshaotong) [inline rpc/src/connection.rs:452]: We can remove this after (https://github.com/labring/sealfs/pull/52) merged.

File: rpc/src/connection.rs [hunk 2]
```diff
@@ -160,7 +158,7 @@ impl ClientConnection {
                     return Err("failed to receive response".into());
                 }
             }
-            sleep(Duration::from_millis(100));
+            // sleep(Duration::from_millis(100));
```
Comment (luanshaotong) [inline rpc/src/connection.rs:161]: We can remove this after (https://github.com/labring/sealfs/pull/52) merged.

File: rpc/src/connection.rs [hunk 3]
```diff
@@ -10,8 +10,6 @@ use log::{debug, error};
 use std::{
     io::{Read, Write},
     sync::{mpsc, Arc, Mutex, RwLock},
-    thread::sleep,
-    time::Duration,
```
Comment (luanshaotong) [inline rpc/src/connection.rs:14]: We can remove this after (https://github.com/labring/sealfs/pull/52) merged.

**Comment by uran0sH** [comment]
Could you use https://github.com/bheisler/criterion.rs to bench the performance?

**Comment by luanshaotong** [comment]
> Could you use https://github.com/bheisler/criterion.rs to bench the performance?

Great suggestion.

**Comment by luanshaotong** [comment]
Can I close this PR？ @Tricky511 @mond77

**Pull request closed without merging**